components/team_sidebar/team_sidebar.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {Team} from 'mattermost-redux/types/teams';
+
+import TeamButton from './team_button';
+
+type Props = {
+    teams: Team[];
+};
+
+type State = {
+
+};
+
+export default class TeamSidebar extends React.PureComponent<Props, State> {
+    renderTeam = (team: Team) => {
+        return (
+            <TeamButton team={team}/>
+        );
+    }
+
+    render() {
+        const {teams} = this.props;
+        const renderedTeams = teams.map(this.renderTeam);
+
+        // TODO temp
+        const root = document.querySelector('#root');
+        if (teams.length <= 1) {
+            root!.classList.remove('multi-teams');
+            return null;
+        }
+        root!.classList.add('multi-teams');
```

Comment:
The class changes should happen in `componentDidMount`/`componentDidUpdate` and only when the number of teams changes

Reply:
Oh, this is how it works currently.... Gross.

Well, we can fix this when we get to actually rewriting this.

Reply:
Yeah, this sidebar isn't even in use right now anyways.